Final pre-release changes (#47)
devfile/registry-operator: DEVFILE_REGISTRY.md

# Devfile Registry

> DevfileRegistry is a custom resource allows you to create and manage your own index server and registry viewer. In order to be added, the Devfile 

Reviewer: ```suggestion
DevfileRegistry is a custom resource allowing you to create and manage your own index server and registry viewer. In order to be added, the Devfile 
```

Author: Revisions under https://github.com/devfile/registry-operator/pull/47/commits/0b7c7c22e0f349498a1741b8452671b023c4cfe4

> Registry must be reachable, supports the Devfile v2.0 spec and above, and is not using the default namespace.

Reviewer: ```suggestion
Registry must be reachable, must support the Devfile v2.0 spec and above, and cannot be deployed to the default namespace.
```

Author: Revisions under https://github.com/devfile/registry-operator/pull/47/commits/0b7c7c22e0f349498a1741b8452671b023c4cfe4
